packages/@okta/vuepress-site/docs/guides/terraform-create-custom-auth-server/main/index.md

```diff
@@ -0,0 +1,396 @@
+
+1. When Okta runs this rule as an optional advanced feature, it can contact an external system to enrich the newly minted token with another type of token from the external system. This is called an *inline hook*. To do this, you need to create another Terraform resource of type `inline_hook` and then set `inline_hook_id` to the ID of that resource. See the [provider documentation for inline hooks](https://registry.terraform.io/providers/okta/okta/latest/docs/resources/inline_hook).
+
+1. If this is your only authorization server policy rule, set `priority` to `1`. For more than one rule, set the `priority` attribute to `1`, `2`, `3`, and so on based on the desired order. To ensure reliable deployment of multiple rules, you must force the Terraform resource creation order to match the `priority` attribute.  Add the `depends_on` Terraform meta attribute to the rule with priority `2` so it depends on the one with priority `1`, and so on.  For more information, see [Managing policy and rule dependencies](/docs/guides/terraform-syntax-tips/main/#managing-policy-and-rule-dependencies).
+
+The following example adds two rules. The first rule grants all claim scopes (plus the default `openid` scope) if the user is in the Okta group defined in Terraform with the name `writable_permissions_api`. The second rule specifies that users in the Okta group with the name `readonly_permissions_api` have access only to the `read_obj` scope (plus the default `openid` scope).
+
+```hcl
+# For users in group writable_permissions_api, grant ALL scopes
+resource "okta_auth_server_policy_rule"
+    "rule_for_staff_with_writable_permissions" {
+  auth_server_id       = okta_auth_server.api_server.id
+  policy_id            = okta_auth_server_policy.example.id
+  status               = "ACTIVE"
+  name                 = "rule_for_staff_with_writable_permissions"
+  priority             = 1
+  group_whitelist      = [ okta_group.writable_permissions_api.id ]
+  grant_type_whitelist = ["authorization_code"]
+  scope_whitelist      = ["*"]
+}
+
+# For users in group readonly_permissions_api, grant ONLY the readapi scope
+resource "okta_auth_server_policy_rule"
+    "rule_for_staff_with_readonly_permissions" {
+  auth_server_id = okta_auth_server.api_server.id
+  policy_id            = okta_auth_server_policy.example.id
+  status               = "ACTIVE"
+  name                 = "rule_for_staff_with_readonly_permissions"
+  priority             = 2
+  group_whitelist      = [ okta_group.readonly_permissions_api.id ]
+  grant_type_whitelist = ["authorization_code"]
+  scope_whitelist      = [ okta_auth_server_scope.read_obj.name , "openid"]
+  depends_on = [
+    okta_auth_server_policy_rule.rule_for_staff_with_writable_permissions ]
+}
+```
+
+Because there's more than one rule, all rules with a `priority` greater than `1` must declare explicit creation order with the `depends_on` meta attribute. See [link to the new syntax article, ideally the heading]
```

Comment:
just double checking this link text?

Reply:
Good catch. I missed that link to an article in the last PR. Fix in next commit.